app/services/food_service.rb

```diff
@@ -0,0 +1,12 @@
+class FoodService 
+    def self.conn 
+        Faraday.new("https://api.nal.usda.gov/fdc/v1/") do |faraday| 
+            faraday.params['api_key'] = ENV['food_api_key']
+        end 
+    end 
+
+    def self.search_food(food)
+        response = conn.get("foods/search?query=#{food}&pageSize=10&pageNumber=1")
```

Comment:
Nice work taking advantage of the pageSize parameter so that you don't have to limit the responses on your end!